js/userDatabase.js

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+const OBJECT_STORE_PREFS = "prefs";
+const PINNED_LINKS_PREF = "pinnedLinks";
+
+(function(exports) {
+  const DATABASE_VERSION = 1;
+  const DATABASE_NAME = "NewTabData";
+
+  const OBJECT_STORE_PREFS_KEY = "prefType";
+  const OBJECT_STORE_PREFS_VALUE = "data";
```

Comment:
same here. It doesn't make much sense to write "OBJECT_STORE_PREFS_VALUE" when you can just write "data"